packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -249,14 +249,14 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
   /**
    * Gets the selection start of the text field
    */
-  public get selectionStart(): number {
+  public get selectionStart(): number | null {
```

Comment:
probably also scary

Reply:
So it already can return null, the typings were just incorrect before. This doesn't change existing behavior, but it does inform partners using strict null checks that they need to do a null check here